src/test/java/kitchenpos/ui/OrderRestControllerTest.java

```diff
@@ -0,0 +1,124 @@
+@SuppressWarnings("NonAsciiCharacters")
+@WebMvcTest(OrderRestController.class)
+class OrderRestControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private OrderService orderService;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @BeforeEach
+    void setUp() {
+    }
+
+    @Nested
+    class 정상_요청_테스트 {
+
+        @Test
+        void 주문_생성() throws Exception {
+            // given
+            Order order = 주문_요청_조리중();
+            given(orderService.create(any()))
+                    .willReturn(order);
+
+            // when & then
+            mockMvc.perform(post("/api/orders")
+                            .contentType("application/json")
+                            .content("{" +
+                                    "\"orderTableId\":1," +
+                                    "\"orderLineItems\":" +
+                                    "[" +
+                                    "{" +
+                                    "\"menuId\":1," +
+                                    "\"quantity\":1" +
+                                    "}" +
+                                    "]" +
+                                    "}"))
+                    .andExpectAll(
+                            status().isCreated(),
+                            header().exists("Location"),
+                            jsonPath("id").value(order.getId()),
+                            jsonPath("orderTableId").value(order.getOrderTableId()),
+                            jsonPath("orderStatus").value(order.getOrderStatus()),
+                            jsonPath("orderLineItems[0].seq").value(order.getOrderLineItems().get(0).getSeq()),
+                            jsonPath("orderLineItems[0].orderId").value(order.getOrderLineItems().get(0).getOrderId()),
+                            jsonPath("orderLineItems[0].menuId").value(order.getOrderLineItems().get(0).getMenuId()),
+                            jsonPath("orderLineItems[0].quantity").value(order.getOrderLineItems().get(0).getQuantity())
+                    );
+        }
+
+        @Test
+        void 주문_목록_조회() throws Exception {
+            // given
+            List<Order> orders = List.of(주문_요청_조리중(), 주문_요청_식사중(), 주문_요청_계산_완료());
+            given(orderService.list())
+                    .willReturn(orders);
+
+            // when & then
+            mockMvc.perform(get("/api/orders"))
+                    .andExpectAll(
+                            status().isOk(),
+                            jsonPath("$").isArray(),
+                            jsonPath("$[0]").exists(),
+                            jsonPath("$[1]").exists(),
+                            jsonPath("$[2]").exists()
+                    );
+        }
+
+        @Test
+        void 주문_상태_수정() throws Exception {
+            // given
+            Order order = 주문_요청_계산_완료();
+            given(orderService.changeOrderStatus(any(), any()))
+                    .willReturn(order);
```

Comment:
```suggestion
            given(orderService.changeOrderStatus(anyLong(), any(Order.class)))
                    .willReturn(order);
```
사소한 것이긴 하지만 이것처럼 구체적으로 any() 메서드를 사용해보시면 어떨까요? 조금 더 가독성을 개선시켜볼 수 있을 것 같아요!

Reply:
리팩터링을 염두에두고 파라미터 타입이 Dto로 변경될 것을 우려하여 `any()`로 놔뒀습니다!
하지만 히이로의 피드백을 듣고보니 되려 테스트 코드를 읽기 어렵게 방해하는 선택이라는 생각이 드는군요!
구체적으로 타입을 명시하겠습니다~!!